internal/git/git.go

```diff
@@ -564,11 +598,17 @@ func userHomeDir() string {
 	}
 }
 
+type Committer struct {
+	Name       string
+	Email      string
+	SigningKey string
```

Comment:
This field is actually no longer used, is it? So all the changes related to `ComitterDetails` could be reverted now that we just use `git` to lookup the data instead of parsing it in as command arguments?